catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/gmti/IndexedMtirpbAttribute.java

```diff
@@ -34,84 +35,91 @@
 
     private static final List<NitfAttribute<TreGroup>> ATTRIBUTES = new LinkedList<>();
 
-    private static final String ATTRIBUTE_NAME_PREFIX = "mtirpb.";
+    private static final String PREFIX = ExtNitfUtility.EXT_NITF_PREFIX + "mtirpb.";
 
+    public static final String INDEXED_TARGET_LOCATION = PREFIX + "target-location";
+
+    public static final String INDEXED_TARGET_CLASSIFICATION_CATEGORY =
+            PREFIX + "target-classification-category";
+
+    public static final String INDEXED_TARGET_AMPLITUDE = PREFIX + "target-amplitude";
+
+    public static final String INDEXED_TARGET_HEADING = PREFIX + "target-heading";
+
+    public static final String INDEXED_TARGET_GROUND_SPEED = PREFIX + "target-ground-speed";
+
+    public static final String INDEXED_TARGET_RADIAL_VELOCITY = PREFIX + "target-radial-velocity";
+
+    public static final String INDEXED_TARGET_LOCATION_ACCURACY =
+            PREFIX + "target-location-accuracy";
     /*
      * Normalized attributes. These taxonomy terms will be duplicated by `ext.nitf.mtirpb.*` when
      * appropriate
      */
 
-    public static final IndexedMtirpbAttribute INDEXED_TARGET_LOCATION = new IndexedMtirpbAttribute(
-            Core.LOCATION,
-            "TGT_LOC",
-            tre -> TreUtility.getTreValue(tre, "TGT_LOC"),
-            new CoreAttributes().getAttributeDescriptor(Core.LOCATION),
-            "targetLocation",
-            ATTRIBUTE_NAME_PREFIX);
+    public static final IndexedMtirpbAttribute INDEXED_TARGET_LOCATION_ATTRIBUTE =
+            new IndexedMtirpbAttribute(Core.LOCATION,
+                    "TGT_LOC",
```

Comment:
@coyotesqrl And another thought, I think we would want to get rid of the short names, since we would want integrators to use only one constant per NITF field, instead of potentially having a mixed batch of them, i.e, a short name and a long name.